prql-compiler/src/ast/item.rs

```diff
@@ -206,21 +206,39 @@ impl Display for Item {
     fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
         match self {
             Item::Ident(s) => {
-                write!(f, "`{s}`")?;
+                fn forbidden_start(c: char) -> bool {
```

Comment:
Should we add a Todo for unifying with the Translator, or is this semantically different?

Reply:
This is different - functionally it's the same, but it translates into different languages, which means it could be different.